avm/res/compute/virtual-machine/tests/e2e/windows.nvidia/main.test.bicep

```diff
@@ -12,7 +12,8 @@ metadata description = 'This instance deploys the module for a VM with dedicated
 param resourceGroupName string = 'dep-${namePrefix}-compute.virtualMachines-${serviceShort}-rg'
 
 @description('Optional. The location to deploy resources to.')
-param resourceLocation string = deployment().location
+#disable-next-line no-unused-params // required because Test-TemplateDeployment.ps1 requires the resourceLocation parameter, value must equal to tempLocation
```

Comment:
Instead I'd actually suggtest to just assign the value to the resourceGroup. Doesn't change the functionality and saves us an extra suppresion :)

Reply:
Done!

Reply:
Done!